src/components/dialogs/parameters/parameters.js

```diff
@@ -612,43 +629,77 @@ const Parameters = ({ user, isParametersOpen, hideParameters }) => {
                             label={<FormattedMessage id="LoadFlow" />}
                             value={TAB_VALUES.lfParamsTabValue}
                         />
-                        <Tab
-                            disabled={!studyUuid}
-                            label={<FormattedMessage id="SecurityAnalysis" />}
-                            value={TAB_VALUES.securityAnalysisParamsTabValue}
-                        />
-                        <Tab
-                            disabled={!studyUuid}
-                            label={
-                                <FormattedMessage id="SensitivityAnalysis" />
-                            }
-                            value={TAB_VALUES.sensitivityAnalysisParamsTabValue}
-                        />
-                        {enableDeveloperMode && (
-                            <Tab
-                                disabled={!studyUuid}
-                                label={<FormattedMessage id="ShortCircuit" />}
-                                value={TAB_VALUES.shortCircuitParamsTabValue}
-                            />
-                        )}
-                        {enableDeveloperMode && (
+                        {!isUnavailableService(
+                            unavailableOptionalServices,
+                            TAB_VALUES.securityAnalysisParamsTabValue
```

Comment:
```suggestion
                            OptionalServicesNames.SecurityAnalysis
```

Reply:
Same for occurrences below, "isUnavailableService" will need, in typescript, an OptionalServicesNames as a parameter, it's more accurate to it this way
Plus, if you change "TAB_VALUES" values, this won't work anymore